### PR TITLE
1385000, update related data

### PR DESCRIPTION
**Pull request by john-par**
Update for task to use .NET tutorial template.

### CODE REVIEW

**Comment by john-par** [comment]
@tdykstra and @Rick-Anderson this PR is ready for your review. Please let me know what you'd like changed.

Please note that a number of the headings in the right ToC still wrap. This is because I couldn't figure out how to  reduce the length of the H2s. If you have any suggestions on ow to shorten them, please let me know. If the H2 lengths for this specific article are acceptable, I am okay with that too.

File: aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application.md [hunk 1]
```diff
@@ -1,24 +1,19 @@
...
+ms.topic: tutorial
 ms.assetid: 7ba88418-5d0a-437d-b6dc-7c3816d4ec07
 msc.legacyurl: /mvc/overview/getting-started/getting-started-with-ef-using-mvc/updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application
 msc.type: authoredcontent
 ---
-Updating Related Data with the Entity Framework in an ASP.NET MVC Application
-====================
-by [Tom Dykstra](https://github.com/tdykstra)
 
-[Download Completed Project](http://code.msdn.microsoft.com/ASPNET-MVC-Application-b01a9fe8)
```
Comment (tdykstra) [inline aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application.md]: I didn't notice before that we were removing these links.  Eventually MSDN code gallery will go away, but until then the links work, and until we find a new home for the sample app code we should leave these links in place.

File: aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application.md [hunk 2]
```diff
@@ -28,6 +23,19 @@ The following illustrations show some of the pages that you'll work with.
...
+> * Add office to the Edit page
+> * Add courses to the Edit page
+> * Update DeleteConfirmed method
+> * Add office location and courses to the Create page
+
+## Prerequisites
+
+* [Reading Related Data](updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application.md)
+
 ## Customize the Create and Edit Pages for Courses
```
Comment (tdykstra) [inline aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application.md]: ## Customize courses pages

File: aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application.md [hunk 3]
```diff
@@ -76,19 +84,20 @@ In *Views\Course\Delete.cshtml* and *Views\Course\Details.cshtml*, change the de
...
-![Course_Index_page_showing_new_course](updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application/_static/image5.png)
-
 Run the **Edit** page (display the Course Index page and click **Edit** on a course).
 
-![Course_edit_page](updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application/_static/image6.png)
-
 Change data on the page and click **Save**. The Course Index page is displayed with the updated course data.
 
-## Adding an Edit Page for Instructors
+## Add office to the Edit page
```
Comment (tdykstra) [inline aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application.md]: ## Add office to instructors page

File: aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application.md [hunk 4]
```diff
@@ -132,13 +141,9 @@ In *Views\Instructor\Edit.cshtml*, after the `div` elements for the **Hire Date*
 
 Run the page (select the **Instructors** tab and then click **Edit** on an instructor). Change the **Office Location** and click **Save**.
 
-![Changing_the_office_location](updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application/_static/image8.png)
-
-## Adding Course Assignments to the Instructor Edit Page
+## Add courses to the Edit page
```
Comment (tdykstra) [inline aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application.md]: Add courses to instructors page

File: aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application.md [hunk 5]
```diff
@@ -198,20 +203,15 @@ Then add a new detail cell immediately following the office location detail cell
...
 
-![Instructor_edit_page_with_courses](updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application/_static/image11.png)
-
 Change some course assignments and click **Save**. The changes you make are reflected on the Index page.
 
  Note: The approach taken here to edit instructor course data works well when there is a limited number of courses. For collections that are much larger, a different UI and a different updating method would be required.
 
-
-## Update the DeleteConfirmed Method
+## Update DeleteConfirmed method
```
Comment (tdykstra) [inline aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/updating-related-data-with-the-entity-framework-in-an-asp-net-mvc-application.md]: Ok to drop "method" if that makes it not wrap

**Comment by john-par** [comment]
@tdykstra your feedback has been incorporated. This PR is ready to be merged.

**Pull request merged**